process/interceptors/processor/chunk/chunk.go

```diff
@@ -0,0 +1,75 @@
+package chunk
+
+import (
+	logger "github.com/ElrondNetwork/elrond-go-logger"
+)
+
+var log = logger.GetOrCreate("process/interceptors/processor")
+
+type chunk struct {
+	maxChunks uint32
+	data      map[uint32][]byte
+	size      int
+}
+
+// NewChunk creates a new chunk instance able to account for the existing and missing chunks of a larger buffer
+// Not a concurrent safe component
+func NewChunk(maxChunks uint32) *chunk {
+	return &chunk{
+		data:      make(map[uint32][]byte),
+		maxChunks: maxChunks,
+	}
+}
+
+// Put will add are rewrite an existing chunk
```

Comment:
`and` instead of `are`

Reply:
changed to `or` as that was the original intent